client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -147,6 +157,7 @@ class ActivityLogDay extends Component {
 						hideRestore={ ! isRewindActive }
 						isDiscarded={ isDiscarded ? isDiscarded( log.activityTs ) : log.activityIsDiscarded }
 						siteId={ siteId }
+						rewindState={ rewindState }
```

Comment:
how about this one? did you decide against using `connect()` in `activity-log-item`?

Reply:
1) Previously you suggested to connect in `ActivityLogDay` see https://github.com/Automattic/wp-calypso/pull/23910#discussion_r179453401

2) Right now it doesn't make sense to me to fire an unknown number of calls, that could be in the hundredths, from ActivityLogItem vs (at most) thirty one calls from ActivityLogDay.

Reply:
I think fetching this from global state is more clear, especially since the activity-log-item component is already connected.

If we're worried about performance, we should profile this, identify the real bottlenecks, and take actions as necessary. For example, we may try to memoize the selector call it actually shows up as a hotspot.

cc'ing @jsnajdr since you've been focusing quite a bit on performance lately. Feel free to chime in if you feel otherwise.

Reply:
I'ts ok, I'll make the change because Dennis pointed in a call that we're going to drop the `ActivityLogDay` soon, so it makes sense to do this change now.

Reply:
The `getRewindState` selector is a simple property getter and doesn't do any computation. The difference between calling it once and calling it 30 times on each Redux state update is likely to be negligible. I'd choose whatever makes the code more readable.